### PR TITLE
Implement CommandOps#getCommand, listCommands

### PR DESCRIPTION
**Pull request by yongpingchen**
- Implement CommandOps#getCommand, listCommands and Small tests
- Large test APIAuthor#postNewCommand, getCommand and listCommands

command APIs of ThingIFAPI will be in another PR

### CODE REVIEW

**Comment by yongpingchen** [comment]
@nfukuzaki can you review this PR?

File: src/ops/CommandOps.ts [hunk 1]
```diff
@@ -74,19 +76,60 @@ export default class CommandOps extends BaseOp {
...
-        //TODO: implement me
-        this.addHeaders({});// add necessary headers
-        return new Promise<Object>((resolve, reject)=>{
-            resolve({});
+    listCommands(listOptions?: ListQueryOptions): Promise<QueryResult<Command>> {
+        return new Promise<QueryResult<Command>>((resolve, reject)=>{
+            var url = this.baseUrl;
+            if(!!listOptions){
+                var queryString = ListQueryOptions.getQueryString(listOptions);
+                if(queryString != ""){
```
Comment (nfukuzaki) [inline src/ops/CommandOps.ts]: I think `if(queryString)` is better.

File: test/small/CommandOpsTest.ts [hunk 2]
```diff
@@ -147,5 +148,281 @@ describe("Test CommandOps", function() {
...
+                        scope = nock(testApp.site, <any>expectedReqHeaders)
+                            .get(path)
+                            .query(test.expectedQueryObject)
+                            .reply(
+                                200,
+                                responseBody,
+                                {"Content-Type": "application/json"}
+                            );
+                        cmdOp.listCommands(new Options.ListQueryOptions(test.bestEffortLimit, test.paginationKey)).then((result: QueryResult<Command>)=>{
+                            expect(result.hasNext).to.be.true;
```
Comment (nfukuzaki) [inline test/small/CommandOpsTest.ts]: please verify `result.paginationKey`

Comment (yongpingchen) [inline test/small/CommandOpsTest.ts]: added

**Pull request merged**